resources/js/pages/game/[game]/requests.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next';
+
+import { SEO } from '@/common/components/SEO';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { GameSetRequestsRoot } from '@/features/games/components/GameSetRequests';
+
+const SetRequestorsPage: AppPage<App.Community.Data.GameClaimsCommentsPageProps> = ({ game }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SEO
+        title={t('List of Set Requests - {{gameTitle}}', { gameTitle: game.title })}
```

Comment:
The page title should reflect the title in the page and the breadcrumb. This should be `'Set Requests - {{gameTitle}}'`.